apis/bigquerydatatransfer/v1alpha1/groupversion_info.go

```diff
@@ -30,4 +30,10 @@ var (
 
 	// AddToScheme adds the types in this group-version to the given scheme.
 	AddToScheme = SchemeBuilder.AddToScheme
+
+	GroupVersionKind = schema.GroupVersionKind{
+		Group:   GroupVersion.Group,
+		Version: GroupVersion.Version,
+		Kind:    "BigQueryDataTransferConfig",
+	}
```

Comment:
A better format could be adding 
`var BigQueryDataTransferConfigGVK = GroupVersion.WithKind("BigQueryDataTransferConfig")` in the <Kind>_types.go, which is what the latest template does.

Reply:
Cool. I am updating the controller template in #2692, so that we don't have a undefined `GroupVersionKind` right after scaffolding the controller code.